### DISCUSSION TITLE
Bigger Preview Window

### DISCUSSION THREAD

**Discussion by johnf** [wbthomason/packer.nvim#972] [Q&A]
Is there a way to make the preview window bigger?

I like to go through the diffs to see if I need to update any configs, but the default preview window takes about 10% of the screen.

**Comment by EdenEast** [comment]
You have two options that handle how the display window is opened. They are `open_fn` and `open_cmd`. The defaults are defined in the [Custom Initialization](https://github.com/wbthomason/packer.nvim#custom-initialization) section.

```lua
require('packer').init({
  display = {
    open_fn  = nil, -- An optional function to open a window for packer's display
    open_cmd = '65vnew \\[packer\\]', -- An optional command to open a window for packer's display
  },
})
```

You can simply make the value `65` bigger to suit your needs or create a function to handle it.

**Reply by johnf** [reply]
That seems to control the width of the packer window itself.
I'm trying to control the split size when you view a diff.
Is that possible?
